plugins/lua/buildingplan/itemselection.lua

```diff
@@ -134,53 +125,71 @@ function ItemSelection:init()
             },
             on_change=self:callback('on_sort'),
         },
-        widgets.HotkeyLabel{
-            frame={l=0, b=1},
-            key='SELECT',
-            label='Use all/none',
-            auto_width=true,
-            on_activate=function() self:toggle_group(self.subviews.flist.list:getSelected()) end,
+        widgets.FilteredList{
+            view_id='flist',
+            frame={t=5, l=1, r=1, b=5},
+            case_sensitive=false,
+            choices=choices,
+            icon_width=2,
+            on_submit=self:callback('toggle_group'),
+            edit_on_char=function(ch) return ch:match('[%l -]') end,
         },
-        widgets.HotkeyLabel{
-            frame={l=22, b=1},
-            key='CUSTOM_SHIFT_C',
-            label='Continue',
-            auto_width=true,
-            on_activate=self:callback('submit'),
+        widgets.Label{
+            frame={l=1, t=5},
+            text_pen=COLOR_LIGHTCYAN,
+            text={">"},
         },
-        widgets.HotkeyLabel{
-            frame={l=38, b=1},
-            key='LEAVESCREEN',
-            label='Go back',
-            auto_width=true,
-            on_activate=self:callback('on_cancel'),
+        widgets.Panel{
+            frame={l=0, t=6, r=0, b=4},
+            frame_style=gui.INTERIOR_FRAME,
         },
         widgets.HotkeyLabel{
-            frame={l=0, b=0},
+            frame={l=0, b=2},
             key='KEYBOARD_CURSOR_RIGHT_FAST',
-            key_sep='    : ',
+            key_sep='ight: ',
             label='Use one',
             auto_width=true,
             on_activate=function() self:increment_group(self.subviews.flist.list:getSelected()) end,
         },
         widgets.Label{
-            frame={l=6, b=0, w=5},
+            frame={l=6, b=2, w=5},
             text_pen=COLOR_LIGHTGREEN,
             text='Right',
         },
         widgets.HotkeyLabel{
-            frame={l=23, b=0},
+            frame={l=1, b=1},
             key='KEYBOARD_CURSOR_LEFT_FAST',
-            key_sep='   : ',
+            key_sep='eft: ',
```

Comment:
is there a functional difference to this change (and the `ight` change above)? These will just get overwritten, correct?

Reply:
yeah, the only "function" of these changes is, like, to let us know if for some reason the label that overwrites these is not working. 🤔 

like, i guess i did this to make it clearer why there were so many spaces in front of that colon. °-°

Reply:
Yeah, an explanatory comment to the right of the line would be a good idea - and also would be a bit more appropriate than a non-functional code change (which could be confusing to future maintainers -- "*should* this be visible???")